io.github.dfandrich.gpscorrelate.json

```diff
@@ -1,19 +1,32 @@
 {
     "app-id":"io.github.dfandrich.gpscorrelate",
     "runtime":"org.freedesktop.Platform",
-    "runtime-version":"22.08",
+    "runtime-version":"23.08",
     "sdk":"org.freedesktop.Sdk",
     "command":"gpscorrelate-gui",
     "finish-args":[
       "--socket=fallback-x11",
       "--socket=wayland",
       "--share=ipc",
-      /* Neeeded until the application uses a file chooser portal */
-      "--filesystem=host",
       /* Needed for gvfs to work */
       "--talk-name=org.gtk.vfs.*"
     ],
     "modules":[
+        {
+          "name": "inih",
```

Comment:
you can just disable INIH when building exiv2. It's only for the command line tool AFAIK.

Reply:
Good to know!